.github/ISSUE_TEMPLATE/stable_feature.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Stable Feature Requirements
+about: Requirements for promoting a feature to stable
+---
+
+This page lists the requirements for promoting a feature to stable. Promotion to stable must meet all requirements of promotion to beta. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+
+**Promotion to Alpha:**
+
+**Promotion to Beta:**
+
+**Relevant Documentation:**
+
+```
+```
+
+--- 
+
+### Requirements: 
+
+**Tests**
+
+- [ ] Automated tests are in place to prevent regressions. 
```

Comment:
I'm not sure how this one is actionable, how would a developer verify this? What is new vs. beta here?